pandas/core/window/rolling.py

```diff
@@ -450,7 +457,36 @@ def homogeneous_func(values: np.ndarray):
             if values.size == 0:
                 return values.copy()
 
+            def _calculate_center_offset(window) -> int:
+                """
+                Calculate an offset necessary to have the window label to be centered.
+
+                Parameters
+                ----------
+                window: ndarray or int
+                    window weights or window
+
+                Returns
+                -------
+                int
+                """
+                if not is_integer(window):
+                    window = len(window)
+                return int((window - 1) / 2.0)
+
+            offset = (
+                _calculate_center_offset(self.window)
+                if self.center
+                and not isinstance(
+                    self._get_window_indexer(self.window), VariableWindowIndexer
```

Comment:
I don't think you should be passing an argument here, just `self._get_window_indexer()`

Reply:
I removed the argument here, too. Thanks for the feedback!